### PR TITLE
ISSUE-182: Make Metadata Display Entities Revisionable

### PR DESCRIPTION
**Pull request by aksm**
Beginning work on #182 (basically just dropped in code from the Drupal docs mentioned in the issue).

### CODE REVIEW

File: format_strawberryfield.install [hunk 1]
```diff
@@ -213,4 +213,111 @@ function format_strawberryfield_update_8702() {
...
+    ];
+
+    return $dependencies;
+  }
+}
+
+/**
+ * Add the 'published' and revisionable metadata fields to my_entity.
+ */
+function my_module_update_8400() {
```
Comment (DiegoPino) [inline format_strawberryfield.install:237]: This function needs to be named format_strawberryfield_ (replace the my_module with this module's name) and the numbering needs to be something after the previous one which was 

`function format_strawberryfield_update_8702() `

So format_strawberryfield_update_8703?

File: format_strawberryfield.install [hunk 2]
```diff
@@ -213,4 +213,111 @@ function format_strawberryfield_update_8702() {
...
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function format_strawberryfield_update_dependencies() {
+  // The update function that adds the status field must run after
+  // content_translation_update_8400() which fixes NULL values for the
+  // 'content_translation_status' field.
+  if (\Drupal::moduleHandler()->moduleExists('content_translation')) {
+    $dependencies['format_strawberryfield'][8400] = [
```
Comment (DiegoPino) [inline format_strawberryfield.install:226]: Same here, 8400 needs to be 8703, etc

File: src/Entity/MetadataDisplayEntity.php [hunk 3]
```diff
@@ -88,12 +88,22 @@
...
+ *   show_revision_ui = TRUE,
  *   admin_permission = "administer metadatadisplay entity",
  *   fieldable = TRUE,
  *   entity_keys = {
  *     "id" = "id",
  *     "label" = "name",
- *     "uuid" = "uuid"
+ *     "uuid" = "uuid",
+ *     "revision" = "revision_id",
+ *     "published" = "status"
```
Comment (DiegoPino) [inline src/Entity/MetadataDisplayEntity.php:101]: 👍

File: src/Entity/MetadataDisplayEntity.php [hunk 4]
```diff
@@ -218,6 +228,17 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDescription(t('The ID of the Metadata Display entity.'))
       ->setReadOnly(TRUE);
 
+    $fields['revision_id'] = BaseFieldDefinition::create('integer')
+      ->setLabel(t('REVISION_ID'))
+      ->setDescription(t('The revision ID of the Metadata Display entity.'))
+      ->setReadOnly(TRUE);
+    $fields['changed'] = BaseFieldDefinition::create('changed')
+      ->setLabel(t('Changed'))
+      ->setDescription(t('The time that the term was last edited.'))
```
Comment (DiegoPino) [inline src/Entity/MetadataDisplayEntity.php:237]: Not a term!

**Review by DiegoPino — CHANGES_REQUESTED** [review]
A few changes that came from the copy

File: src/Entity/MetadataDisplayEntity.php (hunk 4 above)
Comment (DiegoPino) [reply src/Entity/MetadataDisplayEntity.php]: Nothing is stupid, I appreciate the quick response and the learning path you have taken

**Comment by aksm** [comment]
Going to close this and start fresh from 1.0.0 since I have a better understanding of Drupal now.

**Pull request closed without merging**